### PR TITLE
fix(1216): Expressions should be shown similar to data formats in the…

### PR DESCRIPTION
**Pull request by tplevko**
… config panel

fixes https://github.com/KaotoIO/kaoto/issues/1216

Sample running at: https://tplevko.github.io/kaoto-next/

### CODE REVIEW

**Comment by tplevko** [comment]
The second part `Preset "Simple" as the default selected expression language` already done here - https://github.com/KaotoIO/kaoto/pull/1068

File: packages/ui/src/components/Form/expression/ExpressionEditor.test.tsx [hunk 1]
```diff
@@ -1,21 +1,22 @@
 import catalogLibrary from '@kaoto/camel-catalog/index.json';
 import { CatalogLibrary } from '@kaoto/camel-catalog/types';
 import { act, fireEvent, render, screen } from '@testing-library/react';
+import { SchemaService } from '..';
```
Comment (lordrip) [inline packages/ui/src/components/Form/expression/ExpressionEditor.test.tsx:4]:
```suggestion
import { SchemaService } from '../schema.service';
```

**Review by lordrip — APPROVED** [review]
Looks good @tplevko, thanks for taking care. :v:

**Comment by lhein** [comment]
Looks great!

**Pull request merged**